charts/refinery/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: refinery
 description: Chart to deploy Honeycomb Refinery
 type: application
-version: 2.12.3
-appVersion: 2.8.4
+version: 2.12.4
```

Comment:
Let's bump the minor version

Reply:
Is it because this Refinery release is a minor release with bigger changes?